### ISSUE TITLE
CDC COVID-19 - Bug:  VFI for keyboard navigation is either missing or imperceptible

### ISSUE THREAD

**Issue by AccessForAll** [a11ySea/testing#12]
## Issue Summary

The Visual Focus Indicator (VFI) on the CDC site is hard to see on most navigable items; on others, it may be missing or it's imperceptible, especially for people with low-vision.

## Steps to reproduce

1. Go to https://www.cdc.gov/coronavirus/2019-ncov/index.html
2. Press 'tab' to begin navigating through the page.
3. Note the presence of the VFI.
4. When you get to "How to protect yourself" and "What to do if you are sick", note that the VFI is missing (as a low-vision user, I may not be able to see it).

### Expected behavior

The VFI should be visible throughout the whole page.

### Actual behavior

- The VFI is thin and hard to see; it seems to disappear throughout the page, notably on these links:
- "How to protect yourself"
- "What to do if you are sick"
- "Cases in the US"
- "Data and surveillance"
- "Get and keep America open"
- "View more" under "Keeping communities safe" "Latest updates"

### Device Information:

 - OS: Windows 10
 - Browser: Firefox
 - Version: Firefox 75

## Additional References

- Link to any articles or specs that give further guidance
